benchmarks/Elastic.Ingest.Elasticsearch.Benchmarks/Program.cs

```diff
@@ -14,29 +14,8 @@
 using System.Globalization;
 using Elastic.Ingest.Elasticsearch.Benchmarks.Benchmarks;
 
-#if DEBUG
```

Comment:
I found this quite helpful, so I left it in originally.

Reply:
Reverted changes to the benchmark project, wasn't needed in this PR 👍